janusgraph-core/src/main/java/org/janusgraph/core/attribute/Text.java

```diff
@@ -14,75 +14,95 @@
 
 package org.janusgraph.core.attribute;
 
-import com.google.common.base.Preconditions;
-import com.google.common.collect.Sets;
-import org.janusgraph.graphdb.query.JanusGraphPredicate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Set;
+
 import org.apache.commons.lang.StringUtils;
 import org.apache.tinkerpop.gremlin.process.traversal.P;
+import org.janusgraph.graphdb.query.JanusGraphPredicate;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Sets;
 
 /**
- * Comparison relations for text objects. These comparisons are based on a tokenized representation
- * of the text, i.e. the text is considered as a set of word tokens.
+ * Comparison relations for text objects. These comparisons are based on a
```

Comment:
Yes, some of the commits were formatting only.  I can go back through and remove those.